bin/test/i18n/test_i18n_script_utils.rb

```diff
@@ -60,7 +60,7 @@
     let(:to_crowdin_yaml) {described_class.to_crowdin_yaml(to_crowdin_yaml_data)}
 
     let(:to_crowdin_yaml_data) {{en: {'test' => '#example', 'yes' => 'y'}}}
-    let(:to_crowdin_yaml_output) {"---\n:en:\n  test: \"#example\"\n  'yes': 'y'\n"}
+    let(:to_crowdin_yaml_output) {"---\n:en:\n  test: \"#example\"\n  'yes': \"y\"\n"}
```

Comment:
Psych 4 is the YAML parser in Ruby 3.1 and has significantly different behavior (a looooot of the breakage was due to Psych 4), in this case, it returns double quotes rather than single quotes. Change cleared with Artem here: https://github.com/code-dot-org/code-dot-org/pull/60329#issuecomment-2299309326

Reply:
This change to test output is triggered by the Ruby upgrade, so it needs to stay in this branch and can't be split into https://github.com/code-dot-org/code-dot-org/pull/60546